Description: Run MAAS WSGI as non-root
 Run MAAS WSGI as a 'maas' user, fixing the issue of having to
 run various related daemons as root user.
Author: Andres Rodriguez <andreserl@ubuntu.com>
Bug-Ubuntu: http://launchpad.net/bugs/975436

Index: maas-0.1+bzr459+dfsg.orig/contrib/maas-http.conf
===================================================================
--- maas-0.1+bzr459+dfsg.orig.orig/contrib/maas-http.conf	2012-04-12 12:06:44.000000000 -0400
+++ maas-0.1+bzr459+dfsg.orig/contrib/maas-http.conf	2012-04-12 12:17:56.291754257 -0400
@@ -1,6 +1,11 @@
+WSGIDaemonProcess maas user=maas group=maas processes=2 threads=1
 WSGIScriptAlias /MAAS  /usr/share/maas/wsgi.py
 WSGIPassAuthorization On
 
+<Directory /usr/share/maas/>
+    WSGIProcessGroup maas
+</Directory>
+
 <IfModule mod_expires.c>
     <Location /MAAS>
         ExpiresActive On
